<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitCheck</class>
 <widget class="QMainWindow" name="FitCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_6">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="ChckQueries">
                <property name="text">
                 <string>Apply selection queries</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="TxtSelQueries">
                <property name="acceptRichText">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButExportPkl">
                <property name="text">
                 <string>Export *.pkl</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButExportCSV">
                <property name="text">
                 <string>Export *.csv</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0" rowspan="4">
               <widget class="QListWidget" name="LstBoxYPars">
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
                <property name="layoutMode">
                 <enum>QListView::SinglePass</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::ListMode</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="CmbBoxHue"/>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="ButBoxPlot">
                <property name="text">
                 <string>Plot Boxplot</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="CmbBoxX"/>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Hue</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="CmbBoxType"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="4" column="1">
               <widget class="QCheckBox" name="CheckLinesMean">
                <property name="text">
                 <string>Mean</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="ButPlotVect">
                <property name="text">
                 <string>Plot Lines</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" rowspan="6">
               <widget class="QListWidget" name="LstLinesPars">
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
                <property name="layoutMode">
                 <enum>QListView::SinglePass</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::ListMode</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="CmbLinesGroup"/>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Group By</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="CheckLinesStd">
                <property name="text">
                 <string>Std</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="CheckLines">
                <property name="text">
                 <string>Lines</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="TblData">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
